blog/models.py

```diff
@@ -0,0 +1,26 @@
+"""Models for the blog app."""
+
+from django.conf import settings
+from django.db import models
+from django.utils import timezone
+
+
+class Post(models.Model):
+    """A class to represent a post."""
+
+    author = models.ForeignKey(
+        settings.AUTH_USER_MODEL, on_delete=models.CASCADE,
```

Comment:
OTOH, this PR is supposed to follow the tutorial and avoid including arbitrary changes.

Reply:
@webknjaz, sorry, could you, please, explain, why we should avoid arbitrary changes? In my opinion, it's easier just follow the tutorial without the own changes. On the contrary, early I thought that I must modify in my project. I don't mean exactly `authors = models.ManyToManyField`, but in generally.

Reply:
That's because you'd be introducing incompatibilities and deviations that are hard to follow and I already ask to shift the complexity focus towards other development aspects like git, tests, linting, reviews. Once the tutorial is complete and we have testing in place, only then it'd make sense to introduce changes on top.

Reply:
Thank you, I understood

Reply:
Marking as resolved.